data/docmaps/regression_test/docmap_by_manuscript_id/87356.json

```diff
@@ -0,0 +1,705 @@
+                },
+                {
+                    "type": "review-article",
+                    "doi": "10.7554/eLife.87356.2.sa0"
+                },
+                {
+                    "type": "review-article",
+                    "doi": "10.7554/eLife.87356.2.sa1"
+                },
+                {
+                    "type": "evaluation-summary",
+                    "doi": "10.7554/eLife.87356.2.sa2"
+                },
+                {
+                    "type": "reply",
+                    "doi": "10.7554/eLife.87356.2.sa3"
+                }
+            ],
+            "next-step": "_:b6",
+            "previous-step": "_:b4"
+        },
+        "_:b6": {
+            "actions": [
+                {
+                    "participants": [],
+                    "outputs": [
+                        {
+                            "type": "version-of-record",
+                            "identifier": "87356",
+                            "doi": "10.7554/eLife.87356.3",
+                            "versionIdentifier": "3",
+                            "published": "2024-02-14",
+                            "url": "https://doi.org/10.7554/eLife.87356.3",
+                            "content": [
+                                {
+                                    "type": "web-page",
+                                    "url": "https://elifesciences.org/articles/87356"
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ],
+            "assertions": [
+                {
+                    "item": {
+                        "type": "version-of-record",
+                        "doi": "10.7554/eLife.87356.3",
+                        "versionIdentifier": "3"
+                    },
+                    "status": "vor-published"
+                }
+            ],
+            "inputs": [
+                {
+                    "type": "preprint",
+                    "doi": "10.7554/eLife.87356.2",
+                    "identifier": "87356",
+                    "versionIdentifier": "2"
+                }
+            ],
+            "previous-step": "_:b5"
+        },
+        "_:b7": {
+            "actions": [
+                {
+                    "participants": [],
+                    "outputs": [
+                        {
+                            "type": "version-of-record",
+                            "identifier": "87356",
+                            "doi": "10.7554/eLife.87356.3",
+                            "versionIdentifier": "3",
+                            "published": "2024-02-14",
+                            "url": "https://doi.org/10.7554/eLife.87356.3",
+                            "content": [
+                                {
+                                    "type": "web-page",
+                                    "url": "https://elifesciences.org/articles/87356"
```

Comment:
Would it be preferable to use the version URL here (and in steps `_:b6` and `_:b8`)? 

e.g. https://elifesciences.org/articles/87356v1, https://elifesciences.org/articles/87356v2, https://elifesciences.org/articles/87356v3

Reply:
We can do this changes, yes.